daemon/stop.go

```diff
@@ -89,3 +90,17 @@ func (daemon *Daemon) StopPodWithinLock(pod *Pod) (int, string, error) {
 
 	return vmResponse.Code, vmResponse.Cause, nil
 }
+
+func (daemon *Daemon) StopContainer(container string) error {
+	p, idx, ok := daemon.PodList.GetByContainerIdOrName(container)
+	if !ok {
+		return fmt.Errorf("can not found container %s", container)
+	}
+
+	containerId := p.status.Containers[idx].Id
+	glog.V(1).Infof("found container %s to stop", containerId)
+
+	err := p.vm.KillContainer(containerId, syscall.SIGKILL)
+	return err
+	//TODO: update container status of the pod
```

Comment:
Could you update the status of container and keep pod running even container is stopped?